modules/admin_manual/pages/maintenance/manual_upgrade.adoc

```diff
@@ -18,6 +18,12 @@ First, xref:maintenance/backup.adoc[backup] the following items:
 cp -rv /var/www/owncloud /opt/backup/owncloud && mysqldump <db_name> > /opt/backup/backup-file.sql
 ....
 
+[[review-installed-apps]]
```

Comment:
There's no need to explicitly add a header as one, exactly the same, will be auto-generated.

Reply:
I know... for some reason this file already had those so I thought it was an expected pattern

Reply:
also I think it's nice to keep these even when the title changes to make sure that older links still work. or is there another way ?

Reply:
Some do have them, though it's my beliefe that they should be removed. I believe a significant number of them were auto-generated during the conversion from sphinx-doc with Pandoc. There's nothing wrong with keeping them. It just seemed superfluous (in this instance) if one that's exactly the same would be auto-generated.